selenium/src/test/java/au/org/emii/portal/tests/landing/AcknowledgementTest.java

```diff
@@ -0,0 +1,17 @@
+package au.org.emii.portal.tests.landing;
+
+import au.org.emii.portal.tests.BaseTest;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+
+public class AcknowledgementTest extends BaseTest {
+
+    @Test
+    public void linkTest() {
+        webElementUtil.clickLinkContainingText("Acknowledgement");
+        webElementUtil.switchToNewTab();
+        webElementUtil.verifyPageTitle("Data Use Acknowledgement | Portal User Guide");
```

Comment:
The landing page link tests are validating links outside the portal. In PointTimeSeriesLatLonTest, we are using the text "Point timeseries" which can be read from en.js. The drawback will be that it will create a cross dependency of groovy and maven projects (which is not very clean and will increase the complexity). 

Also, I think it is good to keep the two projects independent. This way we have the advantage running the same tests against multiple environment without making config changes. Surely, it will add some extra maintenance. It is almost same like maintaining functional tests.